ext/cl_khr_semaphore.asciidoc

```diff
@@ -430,6 +429,10 @@ being queried by _param_value_. If _param_value_size_ret_ is `NULL`, it is ignor
 | {CL_SEMAPHORE_DEVICE_HANDLE_LIST_KHR}
   | {cl_device_id_TYPE}[]
       | Returns the list of OpenCL devices the semaphore is associated with.
+
+| {CL_SEMAPHORE_IS_EXPORTABLE_KHR}
+    | {cl_bool_TYPE}
+        | Returns CL_TRUE if the semaphore is exportable and CL_FALSE otherwise.
```

Comment:
As discussed in Memory subgroup call on November 14th, move this query to cl_khr_external_semaphore.

Reply:
Done